scripts/update-lockfile.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+make_ignored() {
+  if [ -n "$1" ]
+  then
+    jq keys < lockfile.json | tail --line=+2 | head --lines=-1 | tr -d "\" ," | while read lang
+    do
+      if [ "$lang" != "$1" ]
+      then
+        printf "$lang,"
+      fi
+    done
+  fi
+}
+
+TO_IGNORE=$(make_ignored $1)
+
```

Comment:
You can also use jq to select the string `jq "keys | select(. != $1) | join(\",\")"` ( command composed on mobile) https://stedolan.github.io/jq/manual/#select(boolean_expression)

Reply:
But bash should be fine also

Reply:
As you wish, I am fine with both, but I am definitely more at ease with sh than with jq :wink:

Reply:
You could also add a new env variable for the lua script whatever works best for you.

Reply:
I am fine with keeping the old one, it is sufficient to have the ignore one, so I guess keeping it like this is good.